src/components/Input/MultipleValueTextInput/index.tsx

```diff
@@ -0,0 +1,131 @@
+    onChange?: (values: string[]) => void;
+    placeholder?: string;
+};
+
+export default function MultipleValueTextInput({
+    id,
+    initialValues = [],
+    onChange,
+    placeholder = 'Add value and separate with comma or press enter...',
+}: Props) {
+    const [values, setValues] = useState<string[]>(initialValues);
+    const [inputValue, setInputValue] = useState<string>('');
+    const inputRef = useRef<HTMLInputElement>(null);
+
+    useEffect(() => {
+        setValues(initialValues);
+    }, [initialValues]);
+
+    const handleKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+        const trimmedInput = inputValue.trim();
+        const isEnter = event.key === 'Enter';
+        const isComma = event.key === ',';
+
+        if ((isEnter || isComma) && trimmedInput.length > 0) {
+            event.preventDefault();
+            if (!values.includes(trimmedInput)) {
+                const newValues = [...values, trimmedInput];
+                setValues(newValues);
+                if (onChange) {
+                    onChange(newValues);
+                }
+            }
+            setInputValue('');
+        }
+
+        if (event.key === 'Backspace' && trimmedInput.length === 0) {
+            event.preventDefault();
+            const newValues = values.slice(0, -1);
+            setValues(newValues);
+            if (onChange) {
+                onChange(newValues);
+            }
+            if (inputRef.current) {
+                inputRef.current.focus();
+            }
+        }
+    };
+
+    const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const val = event.target.value;
+        if (val.endsWith(',')) {
+            const trimmedVal = val.slice(0, -1).trim();
+            if (trimmedVal.length > 0 && !values.includes(trimmedVal)) {
+                const newValues = [...values, trimmedVal];
+                setValues(newValues);
+                if (onChange) {
+                    onChange(newValues);
+                }
+            }
+            setInputValue('');
+        } else {
+            setInputValue(val);
+        }
+    };
+
+    const handleRemoveTag = (valueToRemove: string) => {
+        const newValues = values.filter((value) => value !== valueToRemove);
+        setValues(newValues);
+        if (onChange) {
+            onChange(newValues);
+        }
+        if (inputRef.current) {
+            inputRef.current.focus();
+        }
+    };
+
+    return (
+        <div
+            className={styles.container}
+            onClick={() => inputRef.current?.focus()}
+            onKeyDown={(event) => {
+                if (event.key === 'Enter' || event.key === ' ') {
+                    inputRef.current?.focus();
+                }
+            }}
+            tabIndex={0}
+        >
+            {values.map((value) => (
+                <span key={value} className={styles.tag}>
+                    {value}
+                    <button
```

Comment:
Can we make X to remove item from multiple values text input little bit bigger in same style as in multi select input for example used in filters in inventories for enum property (e.g. Property filter State in certificates inventory)?